example-junit5/pom.xml

```diff
@@ -14,9 +14,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.epam.reportportal</groupId>
+            <groupId>com.github.reportportal</groupId>
             <artifactId>agent-java-junit5</artifactId>
-            <version>5.0.0-BETA-2</version>
+            <version>8b159a</version>
```

Comment:
Guess that too outdated, please update dependencies.

Reply:
updated